articles/custom-domains/index.md

```diff
@@ -96,94 +97,14 @@ If you are unable to complete the verification process within three days, you'll
-
-```js
-var lock = new Auth0Lock(config.clientID, config.auth0Domain, {
-...
-	configurationBaseUrl: config.clientConfigurationBaseUrl,
-	overrides: {
-		__tenant: config.auth0Tenant,
-		__token_issuer: config.auth0Domain
-	},
-...
-});
-```
-
-#### Embedded Lock
-
-If you're using Embedded Lock (Lock v11), you need to use your custom domain when initializing Lock. You will also need to set the **configurationBaseUrl** to the appropriate CDN URL:
-
-```js
-var lock = new Auth0Lock('your-client-id', 'login.northwind.com', {
-...
-    configurationBaseUrl: 'https://cdn.auth0.com'
-...
-});
-```
-
-:::note
-The CDN URL varies by region. For regions outside of the US, use `https://cdn.{region}.auth0.com`.
-:::
-
-#### SDKs
-
-If you are using [Auth0.js](/libraries/auth0js) or other SDKs, you will have to initialize the SDK using your custom domain. For example, if you are using the auth0.js SDK, you'll need to set the following:
-
-```js
-webAuth = new auth0.WebAuth({
-  domain:       'login.northwind.com',
-  clientID:     'your-client-id'
-});
-```
-
-Note that the Management API only accepts Auth0 domains. If you use a [custom domain](/custom-domains) and also intend to perform [Management API actions with Auth0.js](/libraries/auth0js/v9#user-management), you must instantiate a new copy of `webAuth` using your Auth0 domain.
-
-#### Auth0 emails
-
-If you would like your custom domain used with your Auth0 emails, you'll need to enable this feature in the [Dashboard](${manage_url}/#/tenant). You can do this by clicking the toggle associated with the **Use Custom Domain in Emails**. When the toggle is green, this feature is enabled.
-
-![](/media/articles/custom-domains/cd_email_toggle.png)
-
-#### Social identity provider configuration
-
-If you want to use social identity providers with your custom domain, you must update the allowed callback URLs to include your custom domain (such as `https://login.northwind.com/login/callback`).
-
-:::warning
-You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
-:::
-
-#### APIs
-
-If you are using Auth0 with a custom domain to issue Access Tokens for your APIs, then you must validate the JWT issuer(s) against your custom domains. For example, if using the [express-jwt](https://github.com/auth0/express-jwt) middleware:
-
-```js
-app.use(jwt({ 
-	issuer: 'https://login.northwind.com', 
-	... additional params ...
-}));
-```
-
-:::note
-If you are using built-in Auth0 APIs, such as the Management API, the API identifier will use your default tenant domain name (such as `https://northwind.auth0.com/userinfo** and **https://northwind.auth0.com/api/v2/`)
-:::
+There may be additional steps you must complete depending on which Auth0 features you are using. Take a look at our [feature-specific setup guide](/custom-domains/feature-specific-setup) for more information.
```

Comment:
when you change the new docs name, change this link as well

Reply:
I'd also rename this section to sth simpler like "Additional configuration steps"